### PR TITLE
Add `django_celeryd_timeoutsec` variable

### PR DESCRIPTION
**Pull request by DavisRayM**
# Changes

- Add ability to configure how long systemd should wait till it times out while trying to start the celeryd service

_In some deployments the ability to configure the TimeoutStartSec and increase it from the default `100` seconds is a nice to have_

### CODE REVIEW

**Review by denniswambua — APPROVED** [review]

File: templates/etc/systemd/celeryd.service.j2 [hunk 1]
```diff
@@ -9,6 +9,7 @@ Group={{ django_system_group }}
 EnvironmentFile=-/etc/default/celeryd-{{ django_system_user }}
 WorkingDirectory={{ django_codebase_path }}
 RuntimeDirectory=celery-{{ django_system_user }}
+TimeoutStartSec= {{ django_celeryd_timeoutsec }}
```
Comment (moshthepitt) [inline templates/etc/systemd/celeryd.service.j2:12]: Nitpick: the empty space after equals bothers me because nothing else is like that in this file :)

Comment (DavisRayM) [inline templates/etc/systemd/celeryd.service.j2:12]:
```suggestion
TimeoutStartSec={{ django_celeryd_timeoutsec }}
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   test:
     name: Test
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```
Comment (moshthepitt) [inline .github/workflows/ci.yml:10]: Should we be concerned that we aren't testing against newer versions of Ubuntu?

Comment (DavisRayM) [reply .github/workflows/ci.yml:10]: [This](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on) doesn't really test whether the role can run on the particular Ubuntu version. Usually just specifies what image should be used to run the molecule test; We'd need to change the image [here](https://github.com/onaio/ansible-django/blob/master/molecule/default/molecule.yml#L8) if we'd like to test using something above 18.04....

_Main reason for this change is an issue with systemd running on the ubuntu-latest image_

File: templates/etc/systemd/celeryd.service.j2 (hunk 1 above)
Comment (DavisRayM) [reply templates/etc/systemd/celeryd.service.j2]: Resolved in latest commits

**Review by moshthepitt — APPROVED** [review]

**Pull request merged**